tasks/generate_mapping.py

```diff
@@ -235,11 +235,13 @@ def _write_attached_storage_domains(f, dc_service, dc):
                 f.write("  dr_primary_address: %s\n" %
                         attached_sd.storage.volume_group
                         .logical_units[0].address)
-                targets = ''
+                targets = set()
```

Comment:
targets = set(lun_unit.target for lun_unit in attached_sd.storage.volume_group.logical_units)

Reply:
Thanks, will do